### DISCUSSION TITLE
Downloads stopped working

### DISCUSSION THREAD

**Discussion by rkrehn** [rkrehn/YTAD#7] [Announcements]
I was just made aware that downloads have stopped working. I'm looking into this issue.

**Comment by rkrehn** [comment]
VideoController needs to be updated (see https://github.com/Tyrrrz/YoutubeExplode/pull/818/files). Will submit update later today.

**Reply by bounty-torn** [reply]
Thanks for getting on this so quickly!

**Comment by rkrehn** [comment]
Release is out https://github.com/rkrehn/YTAD/releases/tag/1.8
